src/routes/dashboard/sections/splits.section.svelte

```diff
@@ -40,15 +66,18 @@
     ]}
   />
   <div class="content pl-0.5">
-    <SectionSkeleton
-      emptyStateHeadline="No splits"
-      emptyStateEmoji="🫗"
-      emptyStateText="Anyone you split incoming funds with will appear here."
-      loaded={true}
-      empty={splitsData.length === 0}
-    >
-      <SplitsTable splits={splitsData} />
-    </SectionSkeleton>
+    {#key userId}
```

Comment:
what's the key for? if `userId` changes, it would change `splitsTableData`, which would cause the component to re-render anyway?

Reply:
ah i was thinking to show the skeleton's loading state but i guess that's triggered in the getSplits function when setting to undefined before fetching

Reply:
actually, going to keep key for now. skeleton container doesn't reset the height to the initial loading height

Reply:
added a reset to initial height to skeleton when reloading